### ISSUE TITLE
Fallback to ~/.devon/home sets environment variables to wrong IDE installation.

### ISSUE THREAD

**Issue by hohwille** [devonfw/devon-ide#101]
With the latest eclipse and m2e the local repo is not properly set inside eclipse what causes severe issues.
The `$DEVON_IDE_HOME/conf/.m2/settings.xml` contains this line:
```
  <localRepository>${M2_REPO}</localRepository>
```
And the environment variable `M2_REPO` is properly set to `$DEVON_IDE_HOME/conf/.m2/repository`.
When I go to Eclipse preferences to Maven in the `User Settings` tab, I can see that the user settings are properly set (`$DEVON_IDE_HOME/conf/.m2/settings.xml`) but for undefined reasons I can see that `Local Repository` setting is pointing to `/Users/hohwille/.m2/repository` instead of `$DEVON_IDE_HOME/conf/.m2/repository`.

**Comment by hohwille** [comment]
When I open a terminal inside Eclipse (from TM Terminal) and do 
```
echo $M2_REPO
```
I also get the correct repo (`$DEVON_IDE_HOME/conf/.m2/repository`) and not the one in my home directory.
So IMHO a bug in m2e.

**Comment by hohwille** [comment]
If I change the `settings.xml` and hardcode the correct repository path to
```
  <localRepository>/projects/devon/conf/.m2/repository</localRepository>
```
Then after restarting Eclipse everything works correct again. So as it seems the latest version of m2e has a bug resolving environment variables (like `${M2_REPO}`).

**Comment by hohwille** [comment]
https://bugs.eclipse.org/bugs/show_bug.cgi?id=546817

**Comment by hohwille** [comment]
I traced down to the root of the problem. It is not a bug of m2e. The thing is as following:
* `devon` command properly sets `M2_REPO` variable to the exepected value
* for some reason the devon-ide installation I was testing this with was lacking the `scripts/commandlet` folder (because this project started with `oasp4j-ide` and was not properly updated).
* a nice fallback mechanism of the `devon` command in the `devon-ide` now allows to execute commandlets from the `~/.devon/home` fallback what however sets the `DEVON_IDE_HOME` variable to another installation of `devon-ide`.
* then the commandlet is executed in the context of the other IDE so I get my eclipse for the wrong IDE.

Therefore, the actual bug is inside this fallback mechanism. We should instead copy over the `commandlets` from the other IDE but keep `DEVON_IDE_HOME` pointing to the current IDE.

**Comment by hohwille** [comment]
Should be fixed with PR #105
